### DISCUSSION TITLE
microphone cuts out and in

### DISCUSSION THREAD

**Discussion by mrsteve0924** [noisetorch/NoiseTorch#379] [General]
Been using this program with no hiccups for several months now and suddenly it doesn't want to play nice. when i use with teams or zoom my microphone keeps cutting out and sometimes just stops working. if i don't load noisetorch everything works fine.

I've rebuilt noisetorch, zoom and teams. downgraded to last kernel version that it seemed to work with, but still same issue.  I use a virtual camera with teams and zoom most of the time and I had a hunch the issue had something to do with the v4l2loopback module.  But i haven't been able to tie it to that 100%.

Anyone else experiencing similar lately? 

im on kernel 6.1.2

Here is -log output:
2023/01/05 18:15:16 Application starting. Version: 0.12.2 (archlinux)
2023/01/05 18:15:16 CAP_SYS_RESOURCE: true
2023/01/05 18:15:16 Checking if config needs to be initialized
2023/01/05 18:15:16 Couldn't resolve $XDG_CONFIG_HOME falling back to '/root/.config'
2023/01/05 18:15:16 Wrote temp librnnoise to: /tmp/librnnoise-888522855.so
2023/01/05 18:15:16 Couldn't resolve $XDG_CONFIG_HOME falling back to '/root/.config'
2023/01/05 18:15:16 Audioserver package name: pulseaudio
2023/01/05 18:15:16 Audioserver package version: 16.1
2023/01/05 18:15:16 Detected PulseAudio
2023/01/05 18:15:16 couldn't parse server version, regexp didn't match version: 16.1
2023/01/05 18:15:16 Querying pulse rlimit
2023/01/05 18:15:16 Rlimit: {Cur:200000 Max:200000}. Trying to remove.
2023/01/05 18:15:16 Rlimit: {Cur:18446744073709551615 Max:18446744073709551615}
2023/01/05 18:15:16 Loading supressor for pulse
2023/01/05 18:15:16 Loaded null sink as idx: 35
2023/01/05 18:15:16 Loaded ladspa sink as idx: 36
2023/01/05 18:15:16 Loaded loopback as idx: 37
2023/01/05 18:15:16 Loaded remap source as idx: 38
2023/01/05 18:15:16 Deleted temp librnnoise: /tmp/librnnoise-888522855.so

**Comment by mrsteve0924** [comment]
well after many hours of troubleshooting i think i narrowed it down to the teams app. the teams app is from the AUR and it just doesnt play nice with my mic/audio even without noisetorch loaded.  then i tried teams on both chromium browser and the electron app, teams-for-linux, and worked great with noisetorch.

**Reply by ZyanKLee** [reply]
sorry to read that it took so long, which has probably been very frustrating. Though I'm glad you found the issue and a workaround.
